integrations/cache/cache-compat.php

```diff
@@ -68,9 +70,10 @@ public function add_cookie_script() {
 			esc_js( $expiration )
 		);
 
-		$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : ' type="text/javascript"';
-
-		echo "<script{$type_attr}>\n{$js}\n</script>\n"; // phpcs:ignore WordPress.Security.EscapeOutput
+		// Need to register prior to enqueue empty script and add extra code to it.
+		wp_register_script( 'pll_cookie_script', '', array(), POLYLANG_VERSION );
+		wp_enqueue_script( 'pll_cookie_script', '', array(), POLYLANG_VERSION, true );
```

Comment:
You don't need to repeat what you did in `wp_register_script()`.
```suggestion
		wp_enqueue_script( 'pll_cookie_script' );
```